ayon_bin_distro/lakectlpy/wrapper.py

```diff
@@ -111,10 +152,21 @@ def clone_local(
         progress_obj: Optional[Union[worker.BaseProgressItem, worker.ProgressItem]],
         repo_branch_uri: str,
         dist_path: str,
-        exists_okay=False,
+        exists_okay: bool = False,
         print_stdout: bool = False,
     ) -> str:
+        """clone a lakeFs repository to a local path
+
+        Args:
+            exists_okay (): bool delet dist_path if folder exists
```

Comment:
Typo.
```suggestion
            exists_okay (bool): delete dist_path if folder exists
```
Also, the docstring arguments differ in order of the function arguments... needs to be reshuffled.